### PR TITLE
KO : Update debugging-getting-started.md

### PR DESCRIPTION
**Pull request by yeonsuyang**

### CODE REVIEW

**Comment by yeonsuyang** [comment]
@nodejs/nodejs-ko Review please.

**Comment by ghost** [comment]
@nodejs/nodejs-ko, Please have a review, thanks!

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 1]
```diff
@@ -203,6 +208,11 @@ Node 인스펙터에 접속할 수 있는 여러 상용 도구와 오픈소스 
 
 * `Debug` 뷰에서 Node.js 디버그 설정을 실행하거나 `F5` 키를 누르세요.
   [자세한 방법은 여기를 참고하세요.](https://medium.com/gitpod/debugging-node-js-applications-in-theia-76c94c76f0a1)
+  
+### [Eclipse IDE](https://eclipse.org/eclipseide) with Eclipse Wild Web Developer extension
+
+* .js 파일에서 "Debug As... > Node program" 을 선택하거나,
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]:
```suggestion
* .js 파일에서 "Debug As... > Node program"을 선택하거나,
```

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 2]
```diff
@@ -203,6 +208,11 @@ Node 인스펙터에 접속할 수 있는 여러 상용 도구와 오픈소스 
...
+* 실행 중인 Node 애플리케이션(이미`--inspect`로 시작)에 디버거를 연결하는 디버그 설정을 생성하세요.
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]: How about this?
```suggestion
* 실행 중인 (`--inspect`로 시작한) Node 애플리케이션에 디버거를 연결하는 디버그 설정을 생성하세요.
```

File: locale/ko/docs/guides/debugging-getting-started.md [hunk 3]
```diff
@@ -203,6 +208,11 @@ Node 인스펙터에 접속할 수 있는 여러 상용 도구와 오픈소스 
 
 * `Debug` 뷰에서 Node.js 디버그 설정을 실행하거나 `F5` 키를 누르세요.
   [자세한 방법은 여기를 참고하세요.](https://medium.com/gitpod/debugging-node-js-applications-in-theia-76c94c76f0a1)
+  
+### [Eclipse IDE](https://eclipse.org/eclipseide) with Eclipse Wild Web Developer extension
```
Comment (yous) [inline locale/ko/docs/guides/debugging-getting-started.md]: How about this?
```suggestion
### [Eclipse IDE](https://eclipse.org/eclipseide)와 Eclipse Wild Web Developer 확장 프로그램
```

**Review by yous — APPROVED** [review]

**Pull request merged**